srml/staking/src/inflation.rs

```diff
@@ -18,17 +18,19 @@
 
 use sr_primitives::{Perbill, traits::SimpleArithmetic};
 
-/// Linear function truncated to positive part `y = max(0, b [+ or -] a*x)` for PNPoS usage
+/// Linear function truncated to positive part `y = max(0, b [+ or -] a*x)` for PNPoS usage.
```

Comment:
PNPoS = Phragmen Nominated PoS ?

Reply:
it is total payout for nominated PoS
```rust
/// The total payout to all validators (and their nominators) per era.
///
/// Named P_NPoS in the [paper](http://research.web3.foundation/en/latest/polkadot/Token%20Ec
/// onomics/#inflation-model).
```
but I should reorganize with a root doc to introduce notation.